domains/ryzm.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "ryzmae",
+           "email": "juwennagpal24@gmail.com",
+           "discord": "980859238630948886"
+        },
+    
+        "record": {
+            "TXT": "dh=52c6cf67a7ea4b84a67d80a0a94059b4194af15f"
```

Comment:
TXT records alonoe do nothing, discord verification should be in a _discord subdoamin, see https://docs.is-a.dev